tests/testthat/test-assert-that.R

```diff
@@ -0,0 +1,14 @@
+context("assert_that")
+
+test_that("assert_that handles long false assertions gracefully", {
+    expect_error(
+        assert_that(isTRUE(10 + sqrt(25) + sum(1:10) + sqrt(25) + sum(11:20) + sqrt(25) + sum(21:30) + sqrt(25) + sum(31:40) + sqrt(25) + sum(41:50))),
+        "^isTRUE\\(x = 10 \\+ sqrt\\(25\\) \\+ sum\\(1\\:10\\) \\+ sqrt\\(25\\) \\+ sum\\(11\\:20\\) \\+ \\.\\.\\. is not TRUE$"
+    )
+})
+
+test_that("assert_that handles long true assertions gracefully", {
```

Comment:
I don't think you need this test. Doesn't it test the same code as the previous test?

Reply:
No. In the first test, the condition evaluates to FALSE and throws an error.

In this test, I changed the condition slightly so it evaluates to TRUE and passes. I think it's a generally useful habit to get in to test behavior on both side of a fork in the code caused by some condition being `TRUE` / `FALSE`.

Reply:
Oh in that case I think it's even less useful. Why do we expect long successful conditions to behave differently to short successful conditions?

Reply:
This would catch an issue if, for example, someone submitted a PR that tries to format the potential error message before evaluating the condition (causing `deparse()` to return more than one element).

I don't feel THAT strongly about it. If you tell me to drop it, I'll drop it.

Reply:
Then that would be caught by review. I don't think the point of tests is to protect against "attacks" by contributors.

Reply:
I removed the test. This is your package and your call.

But I do respectfully disagree. I'm not talking about malicious attacks by contributors. Well-meaning contributions can still make subtle breaking changes. 

In my opinion, CI is a way to offload to a computer the task of checking all the things that you want to remain true about your code. Keeping all that knowledge in your head opens you up to human error (have you ever reviewed PRs before our first cup of coffee??) and makes it difficult to share review responsibility across multiple reviewers.

Reply:
In my experience, fragile tests of precise error messages have cause many more problems than they solve. 

Adding tests for things that _might_ potentially go wrong is of little value; spend time writing tests for high risk areas that are not already covered.